AUTHORS.rst

```diff
@@ -11,3 +11,4 @@ Contributors
 ------------
 
 * `Tim Cera <https://github.com/timcera>`__: :pull_ogc:`58`
+* `Fernando Aristizabal <https://github.com/fernando-aristizabal>`_
```

Comment:
The RST format for URL is \`text \<url\>\`__ (note the two underscores). Please add the missing underscore.

Reply:
Also, please reference the PR number like the previous contributor: :pull_ogc:\`62\`